docker-compose.local.yml

```diff
@@ -34,14 +45,5 @@ services:
     depends_on:
       - forkpi_proxy
```

Comment:
add `db` here as well

Reply:
@Shrhawk Isn't forkpi_proxy is dependent on forkpi which depends on dp..so indirectly cron is dependent on db?
Please correct me if I am wrong